client/web/src/utils/audioUtils.js

```diff
@@ -71,9 +69,9 @@ export const playAudios = async (
     );
     const bs = audioContextRef.current.createBufferSource();
     bs.buffer = audioBuffer;
-    bs.connect(audioSourceNodeRef.current);
-    setupAvatarLipSync(audioContextRef.current, bs);
-    handleFirstInteractionAudio(); // For the first interaction, we need to play a sound to unlock blend shapes
+    // bs.connect(audioSourceNodeRef.current);
```

Comment:
do u still need this?

Reply:
Wait, need it. We thought it would affect the audio to be double audio. However, I just realised that it won't work on other no Avatar Characters.

Reply:
I will take a quick fix on it!

Reply:
All right, fix it